crates/fluvio-cli/src/client/topic/list.rs

```diff
@@ -120,11 +146,45 @@ mod display {
                                 .map(|d| d.filter.transform.uses.as_str())
                                 .unwrap_or("none"),
                         ),
-                        Cell::new(metadata.status.resolution.to_string()),
+                        Cell::new(metadata.status.resolution.resolution_label()),
+                        Cell::new(modified_at.unwrap_or_else(String::new)),
                         Cell::new(metadata.status.reason.to_string()),
                     ])
                 })
                 .collect()
         }
     }
 }
+
+mod utils {
+    use fluvio::{
+        consumer::{ConsumerConfigBuilder, PartitionSelectionStrategy},
+        Fluvio, Offset,
+    };
+    use fluvio_types::Timestamp;
+    use futures_util::stream::StreamExt;
+
+    /// A utility function to get the timestamp of the `Offset::from_end(1)` record across all
+    /// partitions for a given `topic_name`.
+    pub async fn latest_record_timestamp(topic_name: String, fluvio: &Fluvio) -> Option<Timestamp> {
+        let consumer = fluvio
+            .consumer(PartitionSelectionStrategy::All(topic_name))
```

Comment:
This only works for a single partition.  In order to get latest time, need to query all partitions and select latest

Reply:
Hi @sehz, could you clarify this point? [The documentation](https://docs.rs/fluvio/latest/src/fluvio/fluvio.rs.html#220) seems to suggest that this is indeed the way to query all records for a given topic across multiple partitions.

Reply:
[This doc comment](https://docs.rs/fluvio/latest/src/fluvio/consumer.rs.html#591) also corroborates my aforementioned point.

Reply:
But you are only a getting a single record.  You need to get all latest records from all partitions

Reply:
Ahh I see. I just saw the footnote about the lack of ordering guarantees. Thank you!

Reply:
Yes, ordering is guarantee only per partition.  

So if p0 has t1, p2 has t2 and p3 has t3.  you might have any random event with t

Reply:
Hey @sehz, just wanted to make sure I'm on the right track to go about implementing your first suggestion.

As I understand it, we want to extend the [PartitionStatus](https://docs.rs/fluvio/latest/fluvio/metadata/partition/struct.PartitionStatus.html) behavior to allow us to get information about that particular partition's most recent record, is that correct? If that's the case would it be appropriate to give [PartitionStatus](https://docs.rs/fluvio/latest/fluvio/metadata/partition/struct.PartitionStatus.html) access to the [Fluvio](https://docs.rs/fluvio/latest/fluvio/struct.Fluvio.html#) streaming interface by way of something akin to:

```rust
impl PartitionStatus {
  ...
  pub async fn latest_record_timestamp(fluvio: &Fluvio) -> Option<Timestamp> { ... }
  ...
}
```

Also, are there more preferred alternatives to querying records via the consumer API?